### PR TITLE
Milestone 3: Add Contact form

### PR DESCRIPTION
**Pull request by smas0900**
- Added Contact form 
- Linked the form to the formspree

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -334,3 +334,112 @@ body {
...
+  margin-right: 24px;
+}
+
+#fs-frm li {
+  display: flex;
+  flex-direction: column;
+  margin: 0.5em 0;
+}
+
+.Sbtn {
```
Comment (Ismailco) [inline style.css:408]: - [ ] The text is not centered, please make it center and try to duplicate the Figma design as much as possible

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @smas0900,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -334,3 +334,112 @@ body {
   margin-right: 10px;
   margin-left: 10px;
 }
+
+.contact_form {
```
[the remark this answers is not in the document]
Comment (smas0900) [reply style.css:338]: @Ismailco Hi,
can you please elaborate. 
in this comment are you talking about the margin?
 at line 334 margin is for image.

[the remark this answers is not in the document]
Comment (Ismailco) [reply style.css:338]: @smas0900,
I was talking about the padding inside the ```input``` and ``` textarea```, the button text that should be centered, and when you click or hover over the button the colors should be as the Figma design exactly

**Review by gemmen29 — APPROVED** [review]
Hi @smas0900, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions: 
- [ ] This activity requires pair-programming to complete. Kindly invite your partner to participate in your repo.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**